lib/yast2_lan_hostname_base.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: yast2-network
+# Summary: Base test module for all the tests modules of yast2 lan.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+package yast2_lan_hostname_base;
+use parent "y2_module_consoletest";
```

Comment:
Did we check if wicked tests post-fail hook would be useful here?

Reply:
I didn't check that. But these tests also executed on opensuse, where NetworkManager is running, so probably post_fail_hook from wicked will not fit.